From a93a3134ff422af3779eadf724fdef62b399a1c6 Mon Sep 17 00:00:00 2001
From: Meron Cohen <mecohen@habana.ai>
Date: Fri, 09 May 2025 12:22:40 +0300
Subject: [PATCH] [SW-227605] fs: fit_load_type: add missing type

Add missing load type. Although it's only relevant to g2.
FIT_LOAD_TYPE_BIS_E_COMPLIANT_ENABLE is referred from common code.

Change-Id: I730318ea5646e039276c1f4480a3eb4f58726a3c
Signed-off-by: Meron Cohen <mecohen@habana.ai>
---

diff --git a/fs/common/fit_load_type.h b/fs/common/fit_load_type.h
index ddfb2aa..e8a2f56 100644
--- a/fs/common/fit_load_type.h
+++ b/fs/common/fit_load_type.h
@@ -50,6 +50,7 @@
 	FIT_LOAD_TYPE_BIS_D_COMPLIANT_ENABLE = 37,
 	FIT_LOAD_TYPE_READ_LOG = 38,
 	FIT_LOAD_TYPE_WRITE_RACK_EEPROM = 39,
+	FIT_LOAD_TYPE_BIS_E_COMPLIANT_ENABLE = 40,
 	FIT_LOAD_TYPE_COUNT,
 };
 
